analytics/tests/test_counts.py

```diff
@@ -158,7 +159,12 @@ def create_message(self, sender: UserProfile, recipient: Recipient, **kwargs: An
             "content": "hi",
             "date_sent": self.TIME_LAST_HOUR,
             "sending_client": get_client("website"),
+            "realm_id": sender.realm_id,
         }
+        # For simplicity, this helper doesn't support creating cross-realm messages
+        # since it'd require adding an additional realm argument.
+        assert not is_cross_realm_bot_email(sender.delivery_email)
+
         for key, value in defaults.items():
             kwargs[key] = kwargs.get(key, value)
         return Message.objects.create(**kwargs)
```

Comment:
I kinda hate this bundle of functions in the analytics code; it's not worth the refactor as part of this project, but I suspect we'd be happier for them to not do all this model creation by hand.

Reply:
Shall I  open an issue about cleaning this up?

Reply:
Yeah, I'm not sure it's worth much attention at this phase of the release cycle, but it could be worth opening an issue.